extensions/NV/SPV_NV_cluster_acceleration_structure.asciidoc

```diff
@@ -185,6 +186,20 @@ Allowed in *AnyHitKHR* and *ClosestHitKHR* ray tracing execution models.
 |======
 --
 
+(Modify Section 2.16.1, Universal Validation Rules) ::
++
+Modify the list following the statement:
++
+====
+It is invalid for a pointer to be an operand to any instruction other than:
+====
++
+to include:
++
+====
+* <<OpHitObjectGetClusterIdNV,*OpHitObjectGetClusterIdNV*>>
+* <<OpRayQueryGetIntersectionClusterIdNV,*OpRayQueryGetIntersectionClusterIdNV*>>
```

Comment:
spirv.core.grammar.json has this named `OpRayQueryGetClusterIdNV`. Probably worth updating to match.